Documentation/contributing/testing/ci.rst

```diff
@@ -236,19 +236,17 @@ If a specific K8s version fails, it can be re-run using ``retest-X.XX-4.9``.
 
 ``test-missed-k8s`` is deprecated and superseded by ``test-older-k8s``:
 
-- For new branches: it triggers a ``Cilium-PR-Ginkgo-Tests-K8s`` which does
-  exactly the same thing (triggers all ``Cilium-PR-K8s-X.XX-kernel-4.9`` jobs),
-  except that the compound job (and not the individual jobs) will be displayed
-  on the PR page. This makes it more cumbersome to detect which specific K8s
-  version failed in case of failure.
-- For older branches (where ``test-missed-k8s`` is unavailable): triggers a
+- For new branches: it triggers a ``Cilium-PR-Ginkgo-Tests-K8s`` that will
+  automatically abort, warning you that ``test-missed-k8s`` is deprecated and
+  you should use ``test-older-k8s`` instead.
+- For older branches (where ``test-older-k8s`` is unavailable): triggers a
```

Comment:
What is "older" vs. "newer" branches here? Can we simplify this by backporting some change to "older" branches?

Reply:
- Newer = branches that are created after this PR is merged.
- Older = branches that are create before this PR is merged.

I agree it's not very clear...

Regarding backporting changes: actually I think we might be saved by today's decisions regarding which jobs should run on which PR. I have to think a bit about it but I have a hunch we might be able to get rid of `test-missed-k8s` and `test-older-k8s` altogether and replace them with something else.

In any case this PR will probably change quite a bit following the decisions made. I'm switching to draft.